src/btree/bt_sync.c

```diff
@@ -341,6 +350,15 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
                     btree->rec_max_txn = mod->rec_max_txn;
                 if (mod != NULL && btree->rec_max_timestamp < mod->rec_max_timestamp)
                     btree->rec_max_timestamp = mod->rec_max_timestamp;
+
+                /*
+                 * Check whether the tree is lookaside btree and verify whether the page contents
+                 * are obsolete or not by checking the visibility of the page stop time pair.
+                 */
+                if (is_las && __sync_ref_is_obsolete(session, walk))
+                    /* Must be an in-memory leaf page. Evict it. */
+                    WT_ERR(__sync_ref_evict_or_mark_deleted(session, walk));
+
```

Comment:
This isn't necessarily an in-memory leaf page, it's either on-disk or in-memory, isn't it?

Reply:
Yes, Updated the comment to just reflect that it should be a leaf page.